### PR TITLE
PLAT-129635: Migrate storybook to 6

### PR DESCRIPTION
**Pull request by jeongran1230**
**Remove withKnobs options of @storybook/addon-knobs**
- move them to sampler configuration in each theme libraries (see https://www.npmjs.com/package/@storybook/addon-knobs)

**Replace addon-info with addon-docs**
- Remove code related to @storybook/addon-info which is deprecated in storybook 6.0
- Add '@storybook/addon-docs`
- Add @storybook/source-loader in webpack to show source code

**Update dependencies**

Enact-DCO-1.0-Signed-off-by: Jeongran Jang (jeongran.jang@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please update the PR description in detail.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### Changed
+
+- Migrate storybook from 5.3 to 6.1.
+- Replace `@storybook/addon-info` with `@storybook/addon-docs`.
+- Update dependencies for storybook migration
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Migrate storybook from 5.3 to 6.1.
* Replace `@storybook/addon-info` with `@storybook/addon-docs`.
* Update dependencies for storybook migration
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+### Changed
```
Comment (MikyungKim) [inline CHANGELOG.md:1]:
```suggestion
[unreleased]

### Changed
```

File: propTables.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ const removeProps = (component, props) => {
 	}
 
 	props.forEach(prop => {
+		// eslint-disable-next-line react/forbid-foreign-prop-types
```
Comment (MikyungKim) [inline propTables.js:21]: I think we could remove this since `npm run lint` doesn't catch this as a warning.

File: addons/knobs/configure.js [hunk 4]
```diff
@@ -1,11 +1 @@
-import {withKnobs as addonWithKnobs} from '@storybook/addon-knobs';
-
-function withKnobs(opts) {
-	return addonWithKnobs({
-		timestamps: true, // Doesn't emit events while user is typing.
```
Comment (MikyungKim) [inline addons/knobs/configure.js:5]: Don't we need this option anymore?

Comment (jeongran1230) [reply addons/knobs/configure.js:5]: This code is moved to storybook configuration files (@enact/sampler/.storybook/preview.js)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
+## [unreleased]
+
+### Changed
+
+- Migrated storybook from 5.3 to 6.1.
+- Replaced `@storybook/addon-info` with `@storybook/addon-docs`.
+- Updates dependencies for storybook migration
```
Comment (MikyungKim) [inline CHANGELOG.md:7]: Should we change `-` to `*`  for consistency?

**Review by MikyungKim — APPROVED** [review]
LGTM, let's revisit react17 stuff later.

**Pull request merged**